build-ci/jenkins-stack-os-matrix.rst

```diff
@@ -0,0 +1,60 @@
+##############################################################
+Testing the LSST DM Stack with the Jenkins stack-os-matrix Job
+##############################################################
+
+`stack-os-matrix`_ is the continuous integration (CI) job that DM developers must run to ensure tests pass before merging changes to any LSST EUPS Stack package.
```

Comment:
before merging any changes to the *master* branch of any LSST EUPS Stack package.